travis_build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
So far we've been dumping CI-related cruft into the root directory, but it is starting to get a bit busy when Travis uses three different files for itself. We may want to consider starting to shunt things off to a `build/travis/` subdirectory (I believe we can even tuck the main config file in a subdirectory if we also do some configuration through their Web UI).

Reply:
Yes, it would be better to remove some of this cruft from the root directory.